Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.39150&attempt_id=32ce32cd-530a-4af9-8cea-0bc9f6b75b07)

Attempted: rw

<details><summary>Partial log (click to expand)</summary><p>

```
rw.c:281:17: error: use of undeclared identifier 'CLOCK_MONOTONIC'
                clock_gettime(CLOCK_MONOTONIC, &now);
                              ^
rw.c:551:16: error: use of undeclared identifier 'CLOCK_MONOTONIC'
        clock_gettime(CLOCK_MONOTONIC, &start);
                      ^
1 warning and 3 errors generated.
make: *** [<builtin>: rw] Error 1
builder for '/nix/store/nf9sqg96d90fvj1pl8h4g9z2744vs9fm-rw-1.0.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/nf9sqg96d90fvj1pl8h4g9z2744vs9fm-rw-1.0.drv' failed
```
</p></details>

--- a/pkgs/tools/misc/rw/default.nix
+++ b/pkgs/tools/misc/rw/default.nix
@@ -22,5 +22,7 @@ stdenv.mkDerivation rec {
     '';
     license = licenses.isc;
     maintainers = with maintainers; [ dtzWill ];
+
+    broken = stdenv.hostPlatform.isDarwin;
   };
 }

